### DISCUSSION TITLE
Showing Component as Steps

### DISCUSSION THREAD

**Discussion by NessfertIndia** [ant-design-blazor/ant-design-blazor#2290] [Q&A]
Can we show a component as a child element to a Step component? 

```
<Content>
 <Row Justify="center">
    <Col Span="12">
        <Card>
            <Steps ShowProgressDot Current="1" OnChange="onChange">
                <Step Title="Sides" Description="Sides" >
                    <ChooseSides></ChooseSides>
                </Step>
                <Step Title="Personal Info" Description="Basic informations" >

                </Step>
                <Step Title="Review" Description="Review and Confirm" >

                </Step>
            </Steps>
        </Card>
    </Col>         
 </Row> 
</Content>

```

@code {
    int current;

    void onChange(int current)
    {
        this.current = current;
    }

}

This is the error i am getting when i am trying to add Child component ChooseSides as body of step 1 

rror: System.InvalidOperationException: Object of type 'AntDesign.Step' does not have a property matching the name 'ChildContent'.

**Comment by ElderJames** [comment]
You can try `TitleTemple`
